Darwin/Services/StitchRemoteMongoDBService/StitchSyncPerformanceTests/SyncPerformanceIntTestHarness.swift

```diff
@@ -47,7 +44,7 @@ class SyncPerformanceIntTestHarness: BaseStitchIntTestCocoaTouch {
     internal var outputMongoClient: RemoteMongoClient!
     internal var outputColl: RemoteMongoCollection<Document>!
 
-    func setupOutputClient() {
+    func setupOutputClient() -> Bool {
```

Comment:
[style nit] This would probably make more sense as a lazy optional property, and anywhere you access the output client you just guard against if it's `nil`. Slack me if you have any questions